frontend/src/data/variables/AcsPovertyProvider.ts

```diff
@@ -24,9 +24,21 @@ class AcsPovertyProvider extends VariableProvider {
   }
   // ALERT! Make sure you update DataSourceMetadata if you update dataset IDs
   getDatasetId(breakdowns: Breakdowns): string {
+    let datasetPrefix = "acs_poverty_dataset-poverty_by_";
```

Comment:
It's odd to have different variable naming styles (camel case vs underscores) for datasetPrefix and dataset_select.. can you stick with one (preferably the one that is used everywhere else in this file)?